### PR TITLE
style: rename functions

### PR DESCRIPTION
**Pull request by Hao-Chalmers**
### Main improvements in this PR:
* As discussed in #227 , **3** functions were renamed:
    `annotateModel` --> `annotateGEM`
    `importHumanYaml` --> `importYaml`
    `writeHumanYaml`   --> `exportYaml`
* To adapt to above changes, `README` and the functions `increaseHumanGEMVersion`, `testYamlConversion`, `exportHumanGEM` that are in use were modified, while the others (`deleteInactivatedReactions`, `miscModelCurationScript_20190323`, `enableYamlBasedWorkflow`, `removeBoundaryCompartment`, `curateModelECcodes`) were skipped
* Replace `devel` branch with `develop` in the curation workflow, according to `standard-GEM` [v0.5](https://github.com/MetabolicAtlas/standard-GEM/releases/tag/0.5)

**I hereby confirm that I have:**

- [X] Tested my code on my own computer for running the model
- [X] Selected `develop` as a target branch

### CODE REVIEW

**Comment by mihai-sysbio** [comment]
Just to make sure the rename was done everywhere, I've searched for the original function names, with the expectation to find no matches. There were a few results, and at least some of these files are not marked as `.deprecated` from what I can see:

```
Human-GEM on style/renameFunctions took 3s 
❯ grep -inr "importHumanYaml" *
code/modelCuration/deleteInactivatedReactions.m:22:ihuman = importHumanYaml('../../model/Human-GEM.yml');
code/modelCuration/enableYamlBasedWorkflow.m:85:importedHumanGEM = importHumanYaml('testYamlConversion.yml');
code/modelCuration/removeBoundaryCompartment.m:9:ihuman = importHumanYaml('../../model/Human-GEM.yml');
code/modelCuration/curateModelECcodes.m:10:ihuman = importHumanYaml('HumanGEM.yml');

Human-GEM on style/renameFunctions took 3s 
❯ grep -inr "writeHumanYaml" *  
code/modelCuration/deleteInactivatedReactions.m:52:writeHumanYaml(model_del, '../../modelFiles/Human-GEM.yml');
code/modelCuration/miscModelCurationScript_20190323.m:69:writeHumanYaml(ihuman, 'humanGEM');
code/modelCuration/enableYamlBasedWorkflow.m:84:writeHumanYaml(ihuman,'testYamlConversion.yml');
code/modelCuration/removeBoundaryCompartment.m:25:writeHumanYaml(model_del, '../../model/Human-GEM.yml');
code/modelCuration/curateModelECcodes.m:86:writeHumanYaml(ihuman, '../../model/HumanGEM.yml');
```

@Hao-Chalmers  should the renaming also be applied to the files above?

File: README.md [hunk 1]
```diff
@@ -78,8 +78,8 @@ Additional annotation information and external identifiers for Human-GEM reactio
 * Load and save using the built-in MATLAB `load()` and `save()` functions.
 
 `Human-GEM.yml` (Recommended if on `devel` or other branches)
```
Comment (mihai-sysbio) [inline README.md:80]: Shouldn't this be `develop`?

**Review by mihai-sysbio — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Hao-Chalmers) [reply README.md]: nice catch, this is fixed in 0c2f0baf972940984d3ca67092dc14feade63b34

**Review by mihai-sysbio — APPROVED** [review]

**Comment by Hao-Chalmers** [comment]
@mihai-sysbio thanks for the careful check!

Yes, these functions were not included in the renaming because they are one-time scripts and disposable. In my view, they should be moved to `.deprecated` eventually.

**Comment by JonathanRob** [comment]
@mihai-sysbio I haven't yet gone through the `code` folder to move all the deprecated items, but will do so sometime soon.

**Review by JonathanRob — APPROVED** [review]

**Pull request merged**